Storage/Rust/RustRemoteTabs.swift

```diff
@@ -173,4 +211,22 @@ public extension ClientRemoteTabs {
     func toClientAndTabs(client: RemoteClient) -> ClientAndTabs {
         return ClientAndTabs(client: client, tabs: self.remoteTabs.map { $0.toRemoteTab(client: client)}.compactMap { $0 })
     }
+
+    func toClientAndTabs() -> ClientAndTabs {
+        let client = self.toRemoteClient()
+        let tabs = self.remoteTabs.map { $0.toRemoteTab(client: client)}.compactMap { $0 }
+
+        return ClientAndTabs(client: client, tabs: tabs)
+    }
+
+    func toRemoteClient() -> RemoteClient {
+        return RemoteClient(guid: self.clientId,
+                            name: self.clientName,
+                            modified: UInt64(self.lastModified),
+                            type: "\(self.deviceType)",
+                            formfactor: nil,
+                            os: nil,
+                            version: nil,
+                            fxaDeviceId: self.clientId)
```

Comment:
nit: same here for better debugging
```suggestion
        let remoteClient = RemoteClient(guid: self.clientId,
                                        name: self.clientName,
                                        modified: UInt64(self.lastModified),
                                        type: "\(self.deviceType)",
                                        formfactor: nil,
                                        os: nil,
                                        version: nil,
                                        fxaDeviceId: self.clientId)
        return remoteClient
```